src/styles/westpac-payment.scss

```diff
@@ -6,4 +6,29 @@
 
 .westpac-payment-captcha {
     margin: $size-7 0;
-}
\ No newline at end of file
+}
+
+.ob-payment-form__westpac-quickstream-container {
+    border: 1px solid #dedede;
+    border-radius: 2px;
+    background: #fff;
+    padding: 1.5rem;
+}
+
+.ob-payment-form__westpac-quickstream-container .ob-payment-form__westpac-quickstream-amount {
+    padding-bottom: 1.5rem;
+    font-size: 13px;
+    font-weight: 400;
+    color: #2d373e
+}
+
+.quickstream-credit-card-container {
+    container-type: inline-size;
+    container-name: quickstream-credit-card-container;
+}
+
+@container quickstream-credit-card-container (max-width: 767px){
```

Comment:
```suggestion
@container quickstream-credit-card-container (max-width: $mobile){
```

Reply:
767 is the internal westpac breakpoint, so it should stay 767 in my opinion.